lib/instrumentation/modules/apollo-server-core.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const semver = require('semver')
+const shimmer = require('../shimmer')
+
+module.exports = function (apolloServerCore, agent, version, enabled) {
+  if (!enabled) return apolloServerCore
+
+  if (!semver.satisfies(version, '^2.0.0')) {
```

Comment:
The docs and `.tav.yml` use the a `^2.0.2` range, while here it uses `^2.0.0`. This should be `^2.0.2` also, so we don't patch broken/unsupported versions accidentally.

Reply:
```suggestion
  if (!semver.satisfies(version, '^2.0.2')) {
```